cinema/views.py

```diff
@@ -1,45 +1,116 @@
-from rest_framework.decorators import api_view
+from django.http import Http404
+from rest_framework import status, generics, mixins, viewsets
 from rest_framework.response import Response
-from rest_framework import status
+from rest_framework.views import APIView
 
-from django.shortcuts import get_object_or_404
+from cinema.models import Movie, Genre, Actor, CinemaHall
+from cinema.serializers import MovieSerializer, GenreSerializer,\
+    ActorSerializer, CinemaHallSerializer
 
-from cinema.models import Movie
-from cinema.serializers import MovieSerializer
 
-
-@api_view(["GET", "POST"])
-def movie_list(request):
-    if request.method == "GET":
-        movies = Movie.objects.all()
-        serializer = MovieSerializer(movies, many=True)
+class GenreList(APIView):
+    def get(self, request):
+        buses = Genre.objects.all()
```

Comment:
I, think, name "buses" it's from another project) buses don't connect with genre)